src/sagemaker/serve/builder/model_builder.py

```diff
@@ -672,3 +683,51 @@ def validate(self, model_dir: str) -> Type[bool]:
         """
 
         return get_metadata(model_dir)
+
+    def total_inference_model_size_mib(self):
+        """Calculates the model size from HF accelerate
+
+        This function gets the model size from accelerate. It also adds a
+        padding and converts to size MiB. When performing inference, expect
+        to add up to an additional 20% to the given model size as found by EleutherAI.
+        """
+        dtypes = "float32"
```

Comment:
so if no dtype is provided, we assume the worst case scenario of f32 correct?

Reply:
yes